.github/workflows/ci.yml

```diff
@@ -5,34 +5,50 @@ on: [pull_request]
 
 jobs:
   build:
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
+
     strategy:
+      fail-fast: false
       matrix:
-        typo3: [^10.4]
+        typo3: [^9.5, ^10.4]
         php: ['7.2', '7.3', '7.4']
-        os: [ubuntu-latest]
 
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Set up PHP Version ${{ matrix.php }}
-        run: |
-          sudo update-alternatives --set php /usr/bin/php${{ matrix.php }}
-          php -v
+      - name: Setup PHP ${{ matrix.php }}, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: mbstring, dom, zip, mysql
```

Comment:
As we don't have any FunctionalTests here, there is no need to add mysql Extension here